btcstaking-tracker/btcslasher/slasher.go

```diff
@@ -339,3 +257,32 @@ func (bs *BTCSlasher) Stop() error {
 
 	return stopErr
 }
+
+func (bs *BTCSlasher) fetchEvidences() {
+	defer bs.wg.Done()
+	ticker := time.NewTicker(bs.evidenceFetchInterval)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-ticker.C:
+			lastSlashedHeight, err := bs.processEvidencesFromHeight(bs.height)
+			if err != nil {
+				bs.logger.Errorf("error processing evidence from height: %d, err: %v", bs.height, err)
+
+				continue
+			}
+
+			// Only update height if we actually processed evidence
+			if lastSlashedHeight > 0 {
+				bs.mu.Lock()
+				bs.height = lastSlashedHeight + 1
+				bs.mu.Unlock()
+			}
```

Comment:
Does it make sense to set `bs.height` to the current height of Babylon chain if `lastSlashedHeight == 0`?